src/get_account.rs

```diff
@@ -41,9 +41,9 @@ impl ClientCommand for GetAccount {
                 DisplayOrder::BlockIdentifier as usize,
                 true,
             ))
-            .arg(common::public_key::arg(
-                DisplayOrder::PublicKey as usize,
-                PUBLIC_KEY_IS_REQUIRED,
+            .arg(common::account_identifier::arg(
+                DisplayOrder::AccountIdentifier as usize,
+                ACCOUNT_IDENTIFIER_IS_REQUIRED,
             ))
```

Comment:
To retain backwards compatibility at the CLI level:
```suggestion
            .arg(
                common::account_identifier::arg(
                    DisplayOrder::AccountIdentifier as usize,
                    ACCOUNT_IDENTIFIER_IS_REQUIRED,
                )
                .alias(ACCOUNT_IDENTIFIER_ALIAS)
                .short_alias(ACCOUNT_IDENTIFIER_SHORT_ALIAS),
            )
```
where we also have
```rust
const ACCOUNT_IDENTIFIER_ALIAS: &str = "public-key";
const ACCOUNT_IDENTIFIER_SHORT_ALIAS: char = 'p';
```